.golangci.yml

```diff
@@ -9,23 +9,23 @@ run:
 linters:
   disable-all: true
   enable:
-    - prealloc
+    - depguard
+    - exportloopref
+    - gocritic
+    - goimports
+    - gosimple
     - ineffassign
-    - goerr113
     - misspell
-    - errcheck
-    - gosimple
+    - prealloc
     - staticcheck
-    - gosec
-    - gocritic
-    - unparam
-    - deadcode
-    - unconvert
-    - typecheck
     - stylecheck
-    - exportloopref
-    - depguard
-    - goimports
+    - typecheck
+    - unconvert
+    - unparam
+    # TODO: Enable these linters
+    # - errcheck
+    # - goerr113
+    # - gosec
```

Comment:
I have decided to temporarily ignore linters that present numerous errors, seem time-consuming to resolve, or require discussion. This is because I believe it's more important to integrate golangci-lint into GitHub Actions to prevent a continuous increase in errors in the future.

Reply:
got it, thanks 👍